astropy/units/format/console.py

```diff
@@ -36,12 +36,12 @@ def _format_superscript(cls, number):
     @classmethod
     def _format_unit_list(cls, units):
         out = []
-        for base, power in units:
+        for bs, power in units:
```

Comment:
Isn't it customary to avoid name conflicts by using trailing underscores?
```suggestion
        for base_, power in units:
```

Reply:
✅ . Thanks!